logistics_project/apps/malawi/warehouse/runner.py

```diff
@@ -100,6 +100,15 @@ def generate(self, run_record):
                 print "processing hsa %s (%s) (%s of %s)" % (hsa.name, str(hsa.id), i, count)
                 self.update_base_level_data(hsa, start, end, all_products)
 
+        if settings.ENABLE_FACILITY_WORKFLOWS:
+            print 'processing facility data'
+            facilities = SupplyPoint.objects.filter(active=True, type__code=SupplyPointCodes.FACILITY).order_by('id')
+            if self.facility_limit:
```

Comment:
just out of curiosity - is the facility_limit one of the debugging tools used to speed up the runner or does it have another meaning?